src/Cache/Persister/Entity/AbstractEntityPersister.php

```diff
@@ -550,10 +549,8 @@ public function refresh(array $id, object $entity, LockMode|null $lockMode = nul
     }
 
     /** @param array<string, mixed> $ownerId */
-    protected function buildCollectionCacheKey(AssociationMapping $association, array $ownerId, /* string $filterHash */): CollectionCacheKey
+    protected function buildCollectionCacheKey(AssociationMapping $association, array $ownerId, string $filterHash): CollectionCacheKey
```

Comment:
Please document this breaking change in `UPGRADE.md` (extending classes must now also have this argument for this method)